lib/Makefile.am

```diff
@@ -279,7 +280,8 @@ lib_libsyslog_ng_la_SOURCES		= \
 	$(compat_sources)     		\
 	$(logmsg_sources)		\
 	$(str_repr_sources)		\
-	$(logthrsource_sources)
+	$(logthrsource_sources)         \
+        lib/monitor.c
```

Comment:
I think this file is missing (also header).

Reply:
fixed.